coax_client/BlobSequence    sequence
geometry_msgs/Pose2D        pose
